orbit/cmd/orbit/orbit.go

```diff
@@ -861,11 +861,15 @@ func (f *capabilitiesChecker) actor() (func() error, func(error)) {
 // You need to add an explicit check for each capability you want to watch for
 func (f *capabilitiesChecker) execute() error {
 	defer close(f.executeDoneCh)
-	capabilitiesCHeckTicker := time.NewTicker(5 * time.Minute)
+	capabilitiesCheckTicker := time.NewTicker(5 * time.Minute)
+
+	if err := f.client.Ping(); err != nil {
+		log.Error().Err(err).Msg("pinging the server")
```

Comment:
is it okay to just log the error? restarting is also an option

Reply:
I don't know about that, maybe @lucasmrod has some thoughts?

Reply:
Log is a-ok IMO. We don't want to crash Orbit if Fleet is down.

Reply:
It will trigger one restart every time Fleet is down and comes back. But that's probably a-ok for now.